src/js/components/Drop/DropContainer.js

```diff
@@ -51,7 +51,7 @@ class DropContainer extends Component {
     const { onClickOutside } = this.props;
     if (!findDOMNode(this.dropRef).contains(event.target)) {
```

Comment:
I feel we should not call `onClickOutside` if we clicked in the the drop target. Today we are doing that. How about doing something like this:

```
  onClickDocument = (event) => {
    const { dropTarget, onClickOutside } = this.props;
    const dropTargetNode = findDOMNode(dropTarget);
    const dropNode = findDOMNode(this.dropRef);
    if (
      onClickOutside &&
      !dropTargetNode.contains(event.target) &&
      !dropNode.contains(event.target)
    ) {
      onClickOutside();
    }
  }
```

Reply:
@alansouzati thanks for the idea, that makes sense to me.  I will make the change.